app/src/main/res/drawable-ldrtl/onboarding_toolbar_bottom.xml

```diff
@@ -0,0 +1,71 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Sorry just one more nit please add licenses to the two new files top and bottom!

Reply:
Ups, I added it to my file template so I won't always forget. Thanks for pointing it out!